distribution/src/assembly/assembly.xml

```diff
@@ -223,8 +223,16 @@
             <includes>
                 <include>LICENSE</include>
                 <include>NOTICE</include>
+                <include>DISCLAIMER</include>
             </includes>
         </fileSet>
+        <fileSet>
+            <directory>${project.build.directory}</directory>
+            <includes>
+                <include>git.version</include>
```

Comment:
I don't think there's a need to include `git.version` in the binary release. I guess I don't see a reason _not_ to do it, other than it seems a bit 'inside baseball' for a binary release.

Reply:
@gianm yeah, I agree that it might be a bit much for a binary release. I was primarily concerned with being able to differentiate between different RC versions since we lost that ability by not being able to use different versions for RCs. Let's say we roll out an RC2 over a cluster that was running RC1 to fix a bug, and we notice the bug is still happening on one server. It would be harder without versioning information to realize that the actual issue was that the deploy from RC1->RC2 failed on that machine.

Thoughts?

I'm on the fence and would be okay removing it. Maybe it would be better to reduce the amount of info in there to just the commit ID or nearest tag or whatever and bury it away deeper?

Reply:
@drcrallen I was wondering that myself - does the `apache/incubator-druid` repo allow rebase / force pushes?

Reply:
@gianm okay, I decided for the binary, to bury the revision information inside MANIFEST.MF in the JARs. So the manifest file looks like this (bold lines are the ones I added): 

Manifest-Version: 1.0
Implementation-Title: druid-indexing-service
Implementation-Version: 0.13.0-incubating-SNAPSHOT
**Build-Timestamp: 2018-10-17T20:34:23+0000**
Built-By: david
Specification-Vendor: The Apache Software Foundation
Specification-Title: druid-indexing-service
Implementation-Vendor-Id: org.apache.druid
Implementation-Vendor: The Apache Software Foundation
**Build-Revision: `7b51d548afc298b37adf147c5f1eef8c3fcb2f3c`**
**Build-Revision-Describe: druid-0.8.0-rc1-3360-g7b51d54-dirty**
**Build-OS: Mac OS X x86_64 10.13.6**
Created-By: Apache Maven 3.3.3
Specification-Version: 0.13
Implementation-URL: https://druid.apache.org/druid-indexing-service/
**Build-Jdk: 1.8.0_162 (Oracle Corporation 25.162-b12)**
**Build-Version: 0.13.0-incubating-SNAPSHOT**

Does that seem okay to you? I also want to fix `Implementation-URL` since it's getting interpolated to something that doesn't exist.